leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java

```diff
@@ -1,99 +1,105 @@
 package com.schibstedspain.leku.geocoder;
 
+import android.annotation.SuppressLint;
```

Comment:
Is it really needed?

Reply:
Just coz getLastKnownLocation() in new versions of https://github.com/mcharmas/Android-ReactiveLocation library annotated with 
```
    @RequiresPermission(
            anyOf = {"android.permission.ACCESS_COARSE_LOCATION", "android.permission.ACCESS_FINE_LOCATION"}
    )
```

Reply:
We would need to review that. But it's ok for now 👍